### PR TITLE
[langauge] Nop bytecode

### PR DESCRIPTION
**Pull request by dariorussi**
Added the Nop bytecode which may be needed by the compiler/prover

### CODE REVIEW

File: language/tools/vm-genesis/src/genesis_gas_schedule.rs [hunk 1]
```diff
@@ -137,6 +137,7 @@ static INITIAL_GAS_SCHEDULE: Lazy<Vec<u8>> = Lazy::new(|| {
             PackGeneric(StructDefInstantiationIndex::new(0)),
             GasCost::new(73, 1),
         ),
+        (Nop, GasCost::new(10, 1)),
```
Comment (dariorussi) [inline language/tools/vm-genesis/src/genesis_gas_schedule.rs:140]: @tzakian sounds good?

**Comment by dariorussi** [comment]
there is no way to create that from any source language (IR or Move), I assume that is fine

File: language/tools/vm-genesis/src/genesis_gas_schedule.rs (hunk 1 above)
Comment (tzakian) [reply language/tools/vm-genesis/src/genesis_gas_schedule.rs:140]: Looks good to me!

**Comment by dariorussi** [comment]
@tnowacki, @vgao1996, @sblackshear for the Move side of things...

**Comment by shazqadeer** [comment]
Looks good.

**Comment by tnowacki** [comment]
Looks like I missed this before it landed. Could you do me a huge favor and fix the compiler? 
https://github.com/libra/libra/blob/master/language/compiler/ir-to-bytecode/src/compiler.rs#L1665

**Pull request closed without merging**